core/src/datayoga_core/blocks/redis/write/block.py

```diff
@@ -33,7 +33,8 @@ def init(self, context: Optional[Context] = None):
     async def run(self, data: List[Dict[str, Any]]) -> Tuple[List[Dict[str, Any]], List[Result]]:
         pipeline = self.redis_client.pipeline()
         for record in data:
-            dict_as_list = list(reduce(lambda x, y: x + y, record.items()))
+            # transform to a list, filtering it out None, which Redis does not support
+            dict_as_list = sum(filter(lambda i: i[1] is not None and i[0] != Block.MSG_ID_FIELD,record.items()),())
```

Comment:
add spaces:
`dict_as_list = sum(filter(lambda i: i[1] is not None and i[0] != Block.MSG_ID_FIELD, record.items()), ())`